### ISSUE TITLE
Check DDEV environment in settings.php file from Drupal before including the DDEV settings

### ISSUE THREAD

**Issue by jepster** [drud/ddev#2142]
**Is your feature request related to a problem? Please describe.**
The settings.ddev.php file can be deployed on live systems, since it's in the Git repo. Then it's config will be read. Check the DDEV_PROJECT environment variable.

**Describe the solution you'd like**
Modify the auto-generated snippet in the settings.php file:
```
// Automatically generated include for settings managed by ddev.
if (file_exists($app_root . '/' . $site_path . '/settings.ddev.php') && !empty(getenv('DDEV_PROJECT'))) {
  include $app_root . '/' . $site_path . '/settings.ddev.php';
}

```
See the if-case part:
```
!empty(getenv('DDEV_PROJECT'))
```

**Comment by jepster** [comment]
I have seen that this is already implemented in the latest DDEV version (1.13.x).

**Comment by rfay** [comment]
You're FAST! I doesn't update existing settings.php files though, so you can just delete that stanza and then run `ddev start` to get it added again.
